src/child/src/backup.rs

```diff
@@ -0,0 +1,63 @@
+use crate::store::DATA;
+use candid::{candid_method, Decode, Encode};
+use ic_canister_backup::{logic::BACKUP, models::Chunk};
```

Comment:
This `Chunk` type is not consistent with the CLI counterpart

Reply:
yes i changed it